Documentation/gettingstarted/k8s-install-rke.rst

```diff
@@ -0,0 +1,117 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k8s_install_rke:
+
+********************************************
+Installation using Rancher Kubernetes Engine
+********************************************
+
+This guide walks you through integrating Cilium CNI with Kubernetes clusters
+deployed using Rancher Kubernetes Engine (RKE). This guide uses:
+
+  - Rancher Kubernetes Engine v1.0.8
+  - `Helm`_ Version 3
+  - Hosts (1 or more) Running a supported Linux Operating System
+  - `Docker`_
+
+Please consult `RKE Requirements`_ and Cilium :ref:`admin_system_reqs`.
+
+.. _RKE Requirements: https://rancher.com/docs/rke/latest/en/os/
+.. _Docker: https://docker.io
+.. _Helm: https://helm.sh
+
+Installing Rancher Kubernetes Engine
+====================================
+
+Install the RKE Binary according to the `RKE Installation Guide <https://rancher.com/docs/rke/latest/en/installation/>`_
+
+Alternatively, on macOS the binary can be installed using Homebrew:
+
+.. code:: bash
+
+  $ brew install rke
+
+If you have a previous version of RKE installed, you can update it:
+
+
+.. code:: bash
+
+  $ brew upgrade rke
```

Comment:
Homebrew installation is already covered by the official installation guide link above, do we need to duplicate it here?